src/main/java/com/team701/buddymatcher/dtos/pairing/RemoveBuddyDTO.java

```diff
@@ -0,0 +1,28 @@
+package com.team701.buddymatcher.dtos.pairing;
+
+import org.springframework.web.bind.annotation.ModelAttribute;
```

Comment:
Nit: I believe this should be an unused import, I realised looking back at mine, I also have the unused import, but I will submit an issue for removing the unused import in `AddBuddyDTO`

But are you able to take out this unused import